hooks/install-hooks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -eu
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function cp_if_not_exist(){
+  if [ -e "$2" ]
+    then
+      echo "$2 already exists!"
```

Comment:
Should this script support being run multiple times?
For example, I run this script today to setup the checkstyle hook. Then tomorrow someone added a new hook that is useful to me. I want to run both hooks (checkstyle and the new hook that was just added). If I run this script again, it will fail.

Reply:
Well, it would be ideal if this script could support the use case you described. But to do that, the script should be able to not only check duplicate file names, but also the content of those scripts. This can be done by comparing checksums of scripts, but seems over-complicated to me. I guess we can do that later? You can still install any scripts manually.